### DISCUSSION TITLE
Can't center monitor on multiple displays settings

### DISCUSSION THREAD

**Discussion by risalfajar** [linuxdeepin/developer-center#2985] [Features Request & Ideas | 特性请求 & 头脑风暴]
## Describe the bug
I have two monitor which have different sizes, I want my smaller monitor to center aligned with the bigger monitor, but the settings doesn't allow this, my smaller monitor always snapped to bigger monitor's edge.

## To Reproduce
1. Go to 'Multiple Displays' setting
2. Choose 'Customize' and click on 'Custom Settings'
3. Can't center align monitor

## Expected behavior
Monitor should be able to move freely without snapped to the edge, so I can center align them.

## Screenshots
![image](https://user-images.githubusercontent.com/28381124/87236711-b4363f00-c416-11ea-8fcb-6a6d82535cfb.png)

### Distro & Version
Linux Deepin 20 Beta

### Related package version
Deepin Control Center version 5.0.33.0+c7

**Comment by XMuli** [comment]
可能是控制中心尚不支持的功能（后面应该会支持）

**Comment by risalfajar** [comment]
> 可能是控制中心尚不支持的功能（后面应该会支持）

I hope so

**Comment by meiyixiang** [comment]
I've documented this problem, but I'm not sure how difficult this feature is to implement
